cmake/config.cmake

```diff
@@ -269,3 +269,8 @@ set(USE_HEXAGON_SDK /path/to/sdk)
 
 # Whether to use ONNX codegen
 set(USE_TARGET_ONNX OFF)
+
+# Whether to use libbacktrace
+# Libbacktrace provides line and column information on stack traces from errors. It is only
```

Comment:
add an explanation of the exact bug so we know how to sync your patch in the future